doctr/datasets/utils.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import string
+import unicodedata
+import numpy as np
+from typing import Dict
+
+__all__ = ['translate', 'encode_sequence', 'decode_sequence']
+
+
+vocabs: Dict[str, str] = {
+    'digits': string.digits,
+    'ascii_letters': string.ascii_letters,
+    'punctuation': string.punctuation,
+    'currency': '£€¥¢฿',
+    'latin': string.digits + string.ascii_letters + string.punctuation + '°',
+    'french': string.digits + string.ascii_letters + string.punctuation + '°' + 'àâéèêëîïôùûçÀÂÉÈËÎÏÔÙÛÇ' + '£€¥¢฿',
+    'tfrecords_training': '3K}7eé;5àÎYho]QwV6qU~W"XnbBvcADfËmy.9ÔpÛ*{CôïE%M4#ÈR:g@T$x?0î'
+                          '£|za1ù8,OG€P-kçHëÀÂ2É/ûIJ\'j(LNÙFut[)èZs+&°Sd=Ï!<â_Ç>rêi`l'
```

Comment:
I suggest removing this, and keeping only "standard" vocab. This will still be specified in the default config of the model anyway